### PR TITLE
Beginner fixes

### PR DESCRIPTION
**Pull request by gazuntype**
fixes from hafid's beginner tutorial

### CODE REVIEW

File: Editor/DevPanel.cs [hunk 1]
```diff
@@ -241,6 +254,9 @@ public class DevPanel : EditorWindow {
                     _fusionSimulator.bodySimulator.Enable();
                     break;
             }
+
+            _fusionSimulator.faceSimulator.dynamicLightOn = false;
+            _fusionSimulator.bodySimulator.dynamicLightOn = false;
```
Comment (chornesays) [inline Editor/DevPanel.cs:259]: doesnt this turn it off by default? but why?

**Review by chornesays — APPROVED** [review]
Yessssssss

File: Editor/DevPanel.cs (hunk 1 above)
Comment (gazuntype) [reply Editor/DevPanel.cs:259]: it turns off the light sources in the body and face simulators and turns on the one used for fusion.

Comment (chornesays) [reply Editor/DevPanel.cs:259]: oh oh oh oh oh oh oh im dumb merging now

**Pull request merged**